misc/Dockerfile

```diff
@@ -0,0 +1,26 @@
+FROM golang:1.15.4
```

Comment:
most current go version is 1.16.5; go-algorand uses 1.14.7.
Unless we have a good reason to use 1.15.4 here, I'd rather using the same version as in go-algorand for now.

Reply:
This is only used to build indexer, I'm not sure what version of go we use for the indexer releases.

Reply:
yes, I realize that.. using different versions for different products makes it harder to track "which of the known go bugs applies to which product".